### PR TITLE
Fix release CI

### PR DESCRIPTION
**Pull request by fewensa**

### CODE REVIEW

File: .github/workflows/staging.yml [hunk 1]
```diff
@@ -31,6 +31,12 @@ jobs:
         with:
           toolchain: nightly
 
+      - name: Cache target
+        uses: actions/cache@v2
+        with:
+          path: target
```
Comment (AurevoirXavier) [inline .github/workflows/staging.yml:37]: I recommend using a fresh build for release. Any cache might cause a problem.

Is this only for staging?

File: .github/release-template.md [hunk 2]
```diff
@@ -38,5 +38,5 @@ docker pull ghcr.io/darwinia-network/darwinia-2.0:sha-{{ sha }}
 | Pangoro  | `{{ pangoro_proposal_compressed }}`  | `{{ pangoro_runtime_version }}`  |
 
 ## Changelog
-[Darwinia 2.0](https://github.com/darwinia-network/darwinia-2.0/issues/9)
+[Darwinia 2.0](https://github.com/darwinia-network/darwinia/issues/9)
```
Comment (boundless-forest) [inline .github/release-template.md:41]: Please changes this link to https://github.com/darwinia-network/darwinia/pull/969

File: .github/workflows/staging.yml (hunk 1 above)
Comment (fewensa) [reply .github/workflows/staging.yml:37]: Okay, I agree

Comment (fewensa) [reply .github/workflows/staging.yml:37]: What do you think about cache in release CI? If remove it from staging I think also remove it for release CI.

Comment (AurevoirXavier) [reply .github/workflows/staging.yml:37]: You can keep it for staging.

But don't cache it for release.

File: .github/release-template.md (hunk 2 above)
Comment (AurevoirXavier) [reply .github/release-template.md]: Anyway, this will be updated in the final release PR.

**Review by hackfisher — APPROVED** [review]

**Comment by AurevoirXavier** [comment]
Are we waiting for https://github.com/darwinia-network/darwinia/actions/runs/4361727015?

**Comment by fewensa** [comment]
Not need

**Pull request merged**